tools/clang/test/HLSLFileCheck/hlsl/types/struct/bitfields.hlsl

```diff
@@ -0,0 +1,137 @@
+// RUN: %dxc /Zi /T vs_6_2 /E main %s | FileCheck %s
```

Comment:
When using CHECK statements with /Zi you must be careful add a label for the embedded source, otherwise your CHECK statement might match itself from the source instead of the IR you intended to check.
See https://github.com/microsoft/DirectXShaderCompiler/blob/master/tools/clang/test/HLSLFileCheck/Readme.md

Reply:
FWIW, if all your checks use wildcards, and I think yours do, this isn't an issue.